src/Components/LabelSection/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { Badge } from 'antd'
+import './label.css'
+import PropTypes from 'prop-types'
+function LabelSection ({ dataBadge, style, label }) {
```

Comment:
Please change with arrow function.
*based on pattern

Reply:
thankyou for your review, this issue fix in 327bdc89e17135a0db052aba8aa4120c69ceb0f2